### ISSUE TITLE
Fine-tune ARIA live regions in tables with inputs

### ISSUE THREAD

**Issue by hbillings** [18F/cms-hitech-apd#1609]
We've got some tables that have inputs that dynamically change other parts of that table.
For instance:
<img width="604" alt="Screen Shot 2019-06-14 at 3 35 20 PM" src="https://user-images.githubusercontent.com/509309/59539336-0cb8a700-8eba-11e9-81a8-7c3b63de8e86.png">

The relevant quarter `Subtotal` column and the relevant`Total FFP` cell both update any time any of the quarter inputs are updated. The far left bottom cell updates any time any quarter is updated, so cannot be associated with a single live region ID. 

Plus, reading all three of those cells out any time someone changes a percent seems... excessive. But maybe it's helpful? I don't know!

The "advanced" section of [MDN's ARIA live regions page](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions) is helpful, and we may want to use `aria-atomic` and `aria-relevant`.

### This task is done when...
 - [ ] We have a plan for what changes should be announced when.
 - [ ] We have implemented that plan.

**Comment by hbillings** [comment]
After talking with Jerome, the most important thing to announce seems to be the quarterly dollar value when percentages are changed.
